conf/defaults.yaml

```diff
@@ -140,6 +140,14 @@ task.heartbeat.frequency.secs: 3
 task.refresh.poll.secs: 10
 task.credentials.poll.secs: 30
 
+# now should be null by default
+topology.backpressure.enable: true
+backpressure.worker.high.watermark: 0.9
+backpressure.worker.low.watermark: 0.4
+backpressure.executor.high.watermark: 0.9
+backpressure.executor.low.watermark: 0.4
+backpressure.spout.suspend.time.ms: 100
```

Comment:
This seems way too high.  It is not that expensive to poll at a 1ms interval in the spout waiting for back-pressure to be turned off.